### PR TITLE
Secrets - clean some code

### PR DESCRIPTION
**Pull request by yardensade**
## Status
Ready

### CODE REVIEW

File: Tests/scripts/hook_validations/secrets.py [hunk 1]
```diff
@@ -52,6 +50,7 @@
 def get_secrets(branch_name, is_circle):
     secrets_found = {}
     secrets_found_string = ''
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/secrets.py]: remove and switch line 58 to assignment.

File: Tests/scripts/hook_validations/secrets.py [hunk 2]
```diff
@@ -105,23 +94,15 @@ def get_diff_text_files(files_string):
...
-            # R099 TestsPlaybooks/foo.yml TestPlaybooks/playbook-foo.yml
-            # that is why we set index 2 to file_path - the second index is the updated file name
+        if 'r' in file_status.lower():
             file_path = file_data[2]
         else:
             file_path = file_data[1]
-
         # only modified/added file, text readable, exclude white_list file
-        if file_status.upper() in ACCEPTED_FILE_STATUSES and is_text_file(file_path):
+        if (file_status.lower() in ACCEPTED_FILE_STATUSES or 'r' in file_status.lower()) and is_text_file(file_path):
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/secrets.py]: `ACCEPTED_FILE_STATUSES` is uppercase...

File: Tests/scripts/hook_validations/secrets.py [hunk 3]
```diff
@@ -340,6 +320,7 @@ def extract_text_from_pdf(file_path):
         pdf_page = pdf_reader.getPage(page_num)
         page_num += 1
         file_contents += pdf_page.extractText()
+    return file_contents
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/secrets.py]: empty line after condition block

File: Tests/scripts/hook_validations/secrets.py (hunk 2 above)
Comment (yardensade) [reply Tests/scripts/hook_validations/secrets.py]: lol so I didn't change it in the end :s

**Comment by yardensade** [comment]
Wanted to first clear the *NOISE* from the actual changes so it's much more focused and the review process is easier.

**Comment by yardensade** [comment]
Build should be green now

**Comment by yardensade** [comment]
I think we should merge this to master then I will open another pr for part 2

**Pull request merged**